libs/RobotKit/include/StateMachine.h

```diff
@@ -147,6 +150,14 @@ namespace sm::action {
 		auto operator()(irc &rc) const { rc.resetEmergencyStopCounter(); }
 	};
 
+	struct start_choose_autonomous_activity_behavior {
+		auto operator()(irc &rc) const { rc.startChooseAutonomousActivityBehavior(); }
+	};
+
+	struct stop_choose_autonomous_activity_behavior {
+		auto operator()(irc &rc) const { rc.stopChooseAutonomousActivityBehavior(); }
+	};
+
```

Comment:
Est-ce que le choose et le behavior sont importants? Est-ce qu'on ne peut pas avoir plus simplement

```suggestion
	struct start_autonomous_activity {
		auto operator()(irc &rc) const { rc.startAutonomousActivity(); }
	};

	struct stop_autonomous_activity {
		auto operator()(irc &rc) const { rc.stopAutonomousActivity(); }
	};

```